src/menu/columnar_menu.rs

```diff
@@ -299,10 +299,22 @@ impl ColumnarMenu {
         use_ansi_coloring: bool,
     ) -> String {
         if use_ansi_coloring {
-            let match_len = self.working_details.shortest_base_string.len();
+            let match_len = self
+                .working_details
+                .shortest_base_string
+                .trim_end()
```

Comment:
Why do we need to trim the string and thus change the length here?

Reply:
``shortest_base_string`` can include trailing whitespace from the buffer, because the completer will ignore whitespace, when making a completion. "test " -> will complete the same as "test" but will have a different span. so we need to trim it, so 4 chars match and not 5.